runtime/kusama/src/lib.rs

```diff
@@ -1482,8 +1494,9 @@ impl Get<&'static str> for StakingMigrationV11OldPallet {
 }
 
 pub type Migrations = (
+	pallet_balances::migration::ResetInactive<Runtime>,
 	pallet_balances::migration::MigrateToTrackInactive<Runtime, CheckAccount>,
-	crowdloan::migration::MigrateToTrackInactive<Runtime>,
+	crowdloan::migration::MigrateToTrackInactive<Runtime>, //< Must stay in a second time.
```

Comment:
You could rename it to prevent confusion
```suggestion
	crowdloan::migration::MigrateToTrackInactiveV2<Runtime>,
```

Reply:
Actually `pallet_balances::migration::MigrateToTrackInactive<Runtime, xcm_config::CheckAccount>` must stay in a second time as well.